risc0/cargo-risczero/src/lib.rs

```diff
@@ -19,13 +19,12 @@ mod commands;
 mod toolchain;
 mod utils;
 
-#[cfg(feature = "experimental")]
-pub use self::commands::build::BuildSubcommand;
-
 use clap::{Parser, Subcommand};
 
 #[cfg(feature = "experimental")]
 use self::commands::build::BuildCommand;
+#[cfg(feature = "experimental")]
+pub use self::commands::build::BuildSubcommand;
```

Comment:
I like to put re-exports before normal uses so that it's clear what's public.

Reply:
Change reverted in 4d19fe86412bca8d17f87f872c0808d0074511a3